assets/css/pdf.css

```diff
@@ -0,0 +1,21 @@
+@import './AnnotationLayer.css';
+
+.react-pdf__Page {
+    direction: ltr;
+    margin: 1px auto -8px auto;
+    position: relative;
+    overflow: visible;
+    border: 9px solid transparent;
+    background-clip: content-box;
+    border-image: url(../images/shadow.png) 9 9 repeat;
+    background-color: rgba(255, 255, 255, 1);
+}
+.react-pdf__message {
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+}
+.react-pdf__Page__annotations {
+    height: 0;
+}
```

Comment:
Ah is there is no way to pass these styles from the JS? 🤔

Reply:
Can you suggest some way to convert the styles from style.js to a CSS class or passing it to `className` prop directly?

Reply:
React-native-web does not expose any API to convert the styles to a CSS class. Another way would be to get the dom node directly and update the styles onto that but there are many children that are managed by the CSS position over the PDF canvas by the library so our CSS can interfere if we apply CSS as styles. So, at last, we have to add custom classes to the elements which we need to define somewhere. We can't convert the styles to CSS Classes from the existing code so it's better to have a static CSS file. And this file will never change.
we have to include one CSS from the library which enables the Link annotations over PDF. So I think we have to stick with this approach unless you have something in mind.

Reply:
Ok, it sounds easier this way. I don't have any alternative suggestion right now. Just wanted to understand the change as we have basically no CSS files and like to maintain consistency to avoid confusion.